styx-schedule-source/src/main/java/com/spotify/styx/schedule/model/ScheduleDefinition.java

```diff
@@ -24,27 +24,28 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
-import com.spotify.styx.model.DataEndpoint;
+import com.spotify.styx.model.Schedule;
 import java.util.Collections;
 import java.util.List;
 import javax.annotation.Nullable;
 
 /**
- * Mainly a list of {@link DataEndpoint}s
+ * Mainly a list of {@link Schedule}s
  */
 @AutoValue
 @JsonIgnoreProperties(ignoreUnknown = true)
+@Deprecated
 public abstract class ScheduleDefinition {
 
   @JsonProperty
-  public abstract List<DataEndpoint> dataEndpoints();
+  public abstract List<Schedule> schedules();
 
   @JsonCreator
   public static ScheduleDefinition create(
-      @JsonProperty("data_endpoints") @Nullable List<DataEndpoint> dataEndpoints) {
-    if (dataEndpoints == null) {
-      dataEndpoints = Collections.emptyList();
+      @JsonProperty("workflows") @Nullable List<Schedule> schedules) {
+    if (schedules == null) {
+      schedules = Collections.emptyList();
     }
-    return new AutoValue_ScheduleDefinition(dataEndpoints);
+    return new AutoValue_ScheduleDefinition(schedules);
```

Comment:
It would be better to have two models separated and do conversion, so we can easily get rid of the old one later. Like what the other deprecated models. I will give put some conversion methods to help.